src/components/game/index.jsx

```diff
@@ -1,9 +1,57 @@
-import React from "react";
+import React, { useState } from "react";
 import Board from "components/game/Board";
 
 // Renders the Board component
 function Game() {
-  return <Board />;
+  const [valuesArr, setValuesArr] = useState(Array(9).fill(null)); // Values of the 1st-9th square
+  const [nextPlayer, setNextPlayer] = useState("O"); // Initial player aka first player to go is O
+  const winner = detectWinner(); // Checks if there is a winner every render
```

Comment:
This is being computed every `rerender` which is fine in this project but imagine if the board is 100x bigger. This will introduce performance issues. A better way to do this is to turn `winner` into a state and then change it every time `updateValuesArr` is called since that's when it would only change anyway.